index.html

```diff
@@ -73,10 +73,69 @@ <h1 class="worktext">My Recent Works  </h1>
             <li class="langc">javascript</li>
             <li class="langd">ruby</li>
         </ul>
-        <button class="buttonw"><ins class="seep">See Project</ins></button>
+        <button class="buttonw" type="button"><ins class="seep">See Project</ins></button>
     </div>
 </div>
 <!--End Recent of works section-->
+<section class="projectcards">
```

Comment:
- [ ] As the previous reviewer mentioned kindly consider adding the hover event on all the cards, when the user hovers over them it should be a see Project button displayed, this will make your design more professional

![Screenshot_20220903_222538](https://user-images.githubusercontent.com/25227022/188288222-f10dbffc-7807-4c0c-9e7b-383e1b2ee5e4.png)